### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MyPingO**
Added links to the Discord server and the Matrix room

### CODE REVIEW

**Comment by amaanq** [comment]
it'd be nicer to make them badges like [here](https://github.com/tree-sitter/tree-sitter-c)

**Comment by MyPingO** [comment]
Not sure how important the zenodo badge is, so I made it go last

**Comment by amaanq** [comment]
keep it first, and if you're gonna add the npm badges add both the lib and cli and both rust badges.

**Comment by MyPingO** [comment]
I wasn't sure if I should because there are links for them there already. Should I replace the relevant links with badges?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # tree-sitter
 
+[![discord][discord]](https://discord.gg/w7nTvsVJhm)
+[![matrix][matrix]](https://matrix.to/#/#tree-sitter-chat:matrix.org)
+[![npm][npm]](https://www.npmjs.com/package/tree-sitter)
```
Comment (ObserverOfTime) [inline README.md:5]: Should link to the tree-sitter-cli package instead.

**Comment by amaanq** [comment]
those just link to the readme, not registry page. you can just omit the npm package and limit the scope of this pr to just add chat badges

**Comment by MyPingO** [comment]
> those just link to the readme, not registry page. 

The linked readme's have badges to the registry page, wasn't sure if it's better to keep it that way to reduce redundancy, probably doesn't matter either way.

**Comment by amaanq** [comment]
oh right, yeah that's fine as is

**Pull request merged**